### PR TITLE
Functional Tests : `check:step-identifiers` : Improved error message & Added check for baseContext

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | `check:step-identifiers` : Improved error message & Added check for baseContext
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| Related PRs       | N/A
| How to test?      | N/A
| Possible impacts? | N/A

### CODE REVIEW

**Review by nesrineabdmouleh — DISMISSED** [review]

File: tests/UI/tools/stepIdentifiersChecker.js [hunk 1]
```diff
@@ -96,13 +106,70 @@ const checkDoubles = (jsonFile) => {
...
+
+  const baseContextErrors = [];
+  reportContexts.forEach((context) => {
+    const contextFile = context.file.split('/');
+    // Try to rebuild the baseContext from the filename
+    const baseContextFile = contextFile
+      .slice(2, contextFile.length)
+      .map((part) => part
+        .replace(/[0-9]{2}_/, '')
+        .replace('.js', ''))
```
Comment (Progi1984) [inline tests/UI/tools/stepIdentifiersChecker.js]:
```suggestion
        .replace('.js', ''))
        .replace('.ts', ''))
```

Comment (boubkerbribri) [inline tests/UI/tools/stepIdentifiersChecker.js:145]:
```suggestion
        .replace('.js', ''))
        .replace('.ts', ''))
```

**Review by matks — APPROVED** [review]

**Review by matks — CHANGES_REQUESTED** [review]
You have a git conflict :(

**Comment by Progi1984** [comment]
Thanks @matks. I rebased it :)

**Review by matks — APPROVED** [review]

**Review by SD1982 — APPROVED** [review]

**Comment by matks** [comment]
Thank you @Progi1984

**Pull request merged**